src/INode.java

```diff
@@ -461,7 +482,12 @@ private ASTValue evalSaysPredicate(WordsEnvironment environment) {
 
 	private ASTValue evalStatementList(WordsEnvironment environment) {
 		for (int i = 0; i < children.size(); i++) {
-			children.get(i).eval(environment);
+			try {
+				children.get(i).eval(environment);
+			} catch (WordsException e) {
+				System.err.println(e.toString());
+				System.err.print("> ");
```

Comment:
Yes. It's going to print the error, which it would generally do on the same line as the current REPL prompt. The error printing is a little awkward since it comes up to a second after you typed things in.

I'm changing the second line to System.out, since it shouldn't be .err.